doc/src/manual/arrays.md

```diff
@@ -348,7 +348,7 @@ indices and can be converted to such by [`to_indices`](@ref):
 2. An array of scalar indices. This includes:
     * Vectors and multidimensional arrays of integers
     * Empty arrays like `[]`, which select no elements
-    * `Range`s of the form `a:c` or `a:b:c`, which select contiguous or strided subsections from `a` to `c` (inclusive)
+    * `AbstractRange`s of the form `a:c` or `a:b:c`, which select contiguous or strided subsections from `a` to `c` (inclusive)
```

Comment:
I'd simply say: "Ranges like `a:c`…"  Making this more jargon-y implies a distinction that doesn't exist.